patches/chrome-browser-resources-settings-site_settings-category_default_setting.js.patch

```diff
@@ -10,3 +10,11 @@ index 3c75f8db1f1ef32aae97cf3eabb1e4319f121d8d..029ee25931a5914ddc05fb2272d994bb
        case ContentSettingsTypes.BACKGROUND_SYNC:
        case ContentSettingsTypes.IMAGES:
        case ContentSettingsTypes.JAVASCRIPT:
+@@ -194,6 +195,7 @@ Polymer({
+       case ContentSettingsTypes.USB_DEVICES:
+       case ContentSettingsTypes.VR:
+       case ContentSettingsTypes.WINDOW_PLACEMENT:
++      case ContentSettingsTypes.ETHEREUM:
```

Comment:
I'm pretty sure there is another way to do this cc @petemill

Reply:
Right now we don't have an override for ContentSettingsTypes - `AUTOPLAY` is already patched in here and in `site_settings/constants.js`. I have some ideas for how we could extend these instead of patch them. Failing that we could certainly patch in a more extendable way. However, my idea for how to do this is to use a Polymer behavior (i.e. subclass) on `category-default-setting` and override `onChangePermissionControl_`. The issue with this is that behavior additions are currently difficult to achieve in cr93+, so I'd lean to letting this patch go through, especially as it's single line and self-explanatory. If we really want to avoid the patch and move the current `AUTOPLAY` patch, then the behavior is my first idea.

Reply:
```
this.onChangePermissionControl_ = function () {
    if (this.category === undefined ||
        this.controlParams_.value === undefined) {
      // Do nothing unless all dependencies are defined.
      return;
    }
    // Don't override user settings with enforced settings.
    if (this.controlParams_.enforcement ===
        chrome.settingsPrivate.Enforcement.ENFORCED) {
      return;
    }
    if (this.category === ContentSettingsTypes.AUTOPLAY) {
        this.browserProxy.setDefaultValueForContentType(
            this.category,
            this.categoryEnabled ? ContentSetting.ALLOW : ContentSetting.BLOCK);
         return;
    }    
    if (this.category === ContentSettingsTypes.ETHEREUM) {
        this.browserProxy.setDefaultValueForContentType(
            this.category,
            this.categoryEnabled ? ContentSetting.ASK : ContentSetting.BLOCK);
         return;
    }
    old_onChangePermissionControl.apply(this);
}
```
If we do behavior override, probably will be similar as above, most code in the original function will be copied from chromium, as discussed with @petemill in DM, we both think patching is a bit cleaner & easier to maintain in this case.

Reply:
we have a net reduction in patches in this PR so this seems fine then